internal/kafka/internal/handlers/validation.go

```diff
@@ -260,8 +260,9 @@ func ValidateKafkaUpdateFields(kafkaUpdateRequest *private.KafkaUpdateRequest) h
 			stringSet(&kafkaUpdateRequest.KafkaVersion) ||
 			stringSet(&kafkaUpdateRequest.KafkaIbpVersion) ||
 			stringSet(&kafkaUpdateRequest.DeprecatedKafkaStorageSize) ||
-			stringSet(&kafkaUpdateRequest.MaxDataRetentionSize)) {
-			return errors.FieldValidationError("Failed to update Kafka Request. Expecting at least one of the following fields: strimzi_version, kafka_version, kafka_ibp_version, kafka_storage_size or max_data_retention_size to be provided")
+			stringSet(&kafkaUpdateRequest.MaxDataRetentionSize) ||
+			arrays.IsNotNilPredicate(kafkaUpdateRequest.Suspended)) {
```

Comment:
I don't think we want to add this check in here since `suspended` can be omitted from the request?

See thread in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1346#discussion_r995732444

Reply:
I think this is checking that at least one field is present in the update request body: notice the `!` in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/3ab2d9f94366698babaeeebfdccc223f7caa2c6c/internal/kafka/internal/handlers/validation.go#L259

Reply:
this statement checks if any of the parameters is provided and fails if no valid param is present in the body. maybe we should remove it completely and just return 200 in this case?

Reply:
@JameelB No. What I wanted to ask was: "Why are we using array utils predicate functions here?"
Those predicates were meant to be used with the `Find`, `AnyMatch`, `...` functions.

If we believe they can be useful as common utilities, I believe we should move them to another package.

Reply:
for isNil/ isNotNil I think it makes sense to move them to the common utils. should I move it as a part of my PR?